EnviroSense.Web/Controllers/DevicesController.cs

```diff
@@ -115,13 +115,20 @@ public async Task<IActionResult> AddMeasurements(MeasurementViewModel model)
             }
         }
 
-        public async Task<ActionResult> Measurements(Guid deviceId)
+        [HttpGet]
+        public async Task<ActionResult> Measurements(Guid deviceId, int pageIndex)

```

Comment:
```suggestion
        public async Task<ActionResult> Measurements(MeasurementViewModel viewModel)
```

Would it make sense ?

Reply:
I believe this way could help us when we add more filters (later)